### ISSUE TITLE
--mpa-report missing header

### ISSUE THREAD

**Issue by rbutleriii** [DerrickWood/kraken2#327]
When using kraken2 with `--report`, with or without `--use-mpa-style`, is there a way to specify a header line?

**Comment by jenniferlu717** [comment]
No, I believe for both files, there is no default header line. Are you confused by any of the columns?

**Comment by pdhrati02** [comment]
@jenniferlu717 
Hi, I have a similar question. I used kraken2 with --report and --use-mpa-style options for about 96 paired end samples. This generated 2 files (standard output and report file for each sample). However these files have no headers to them, no just for column names but for sample names too. 
I want to combine the reports and process the results further for taxonomy analysis. Can you help me with how to go about this without any sample name headers? Or is there a better way to process kraken2 results?

Any help would be appreciated. 
Thank you
